crates/hdk/src/lib.rs

```diff
@@ -331,7 +331,17 @@ pub mod countersigning;
 /// Most Holochain applications will define their own app entry types.
 ///
 /// App entries are all entries that are not system entries.
-/// They are defined in the `entry_defs` callback and then the application can call CRUD functions with them.
+/// Definitions of entry types belong in the integrity zomes of a DNA. In contrast, operations
```

Comment:
@maackle Is there a way to dedup these doc comments? They need to be updated in two different places, the hdk and the hdi crates.